coral/coral/media/js/views/components/plugins/edit-workflow.js

```diff
@@ -56,6 +56,95 @@ define([
       this.loading(false);
     };
 
+    this.loadAssignConsultationData = async (resourceId) => {
+      const componentData = {};
+      
+      const planningConsultationTiles = await this.fetchTileData(resourceId);
+      this.resourceName(
+        this.getNameFromNodeId(planningConsultationTiles, '18436d9e-c60b-4fb6-ad09-9458e270e993')
+      );
+      for (tile of planningConsultationTiles) {
+        if (tile.nodegroup === 'f5aeaa90-3127-475d-886a-9fc62742de4f'){
+          const planningDigitalFilesTiles = await self.fetchTileData(tile.data['f5aeaa90-3127-475d-886a-9fc62742de4f'][0].resourceId)
+          planningDigitalFilesTiles.forEach((tile) => {
+            componentData[tile.nodegroup] = {
+              value: JSON.stringify({
+                tileData: koMapping.toJSON(tile.data),
+                resourceInstanceId: tile.resourceinstance,
+                tileId: tile.tileid,
+                nodegroupId: tile.nodegroup
+              })
+            };
+
+        })
```

Comment:
@OwenGalvia Please run prettier on this file

Reply:
This is prettied? It's simply a new chunk of code